napari/_qt/layer_controls/qt_tracks_controls.py

```diff
@@ -124,10 +124,20 @@ def _on_head_length_change(self):
     def _on_properties_change(self):
         """Change the properties that can be used to color the tracks."""
         with self.layer.events.properties.blocker():
+            pre_store = self.layer.properties_to_color_by
```

Comment:
`self.layer.properties_to_color_by` returns a list of strings - do we want to store the `self.layer.color_by` string here instead?